pom.xml

```diff
@@ -55,12 +55,12 @@
     <version.com.google.protobuf>3.6.1</version.com.google.protobuf>
     <version.org.eclipse.jdt>3.19.0</version.org.eclipse.jdt>
     <version.com.github.javaparser>3.13.10</version.com.github.javaparser>
-    <version.com.fasterxml.jackson>2.9.10</version.com.fasterxml.jackson>
-    <version.kubernetes-client>4.4.2</version.kubernetes-client>
+    <version.com.fasterxml.jackson>2.10.1</version.com.fasterxml.jackson>
```

Comment:
I am bit worried about changing Jackson version that is then not aligned with Quarkus 
https://github.com/quarkusio/quarkus/blob/1.1.0.Final/bom/runtime/pom.xml#L79
since we rely on it for anything that is based on rest ....

Reply:
I see what you mean here ...

Digging deeper, it seems that kubernetes-client was updated not so long ago with that one so we can fallback to 4.6.2 whcih is still using jackson 2.9.10.
xtf 0.14 is set with kubernetes-client 4.6.1 but is also compatible with that one.